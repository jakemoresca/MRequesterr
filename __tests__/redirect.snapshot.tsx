import { render } from '@testing-library/react'
import Redirect from '../pages/redirect'

it('renders page unchanged', () => {
    global.close = jest.fn();

    const { container } = render(<Redirect />)
    expect(container).toMatchSnapshot()
})

it('must call close window', () => {
    global.close = jest.fn();

    render(<Redirect />);
    expect(global.close).toBeCalled();
})