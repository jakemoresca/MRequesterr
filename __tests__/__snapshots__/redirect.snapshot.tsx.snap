// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders page unchanged 1`] = `<div />`;
